14.0 - MIG - sale order line sequence

patrickt-oforce opened the pull request:
Back port of module `sale_order_line_sequence` to v 14.0

--
Conferm have signed  CLA https://odoo-community.org/page/cla and have read the contributing guideline at https://odoo-community.org/page/contributing

nikul-serpentcs committed:
[12.0] [MIG] sale_order_line_sequence (#744)

* sale_order_line_sequence v10

* OCA Transbot updated translations from Transifex

* Update sale_order_line_sequence.pot

* Translated using Weblate (Portuguese)

Currently translated at 100,0% (6 of 6 strings)

Translation: sale-workflow-10.0/sale-workflow-10.0-sale_order_line_sequence
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-10-0/sale-workflow-10-0-sale_order_line_sequence/pt/

* sale_order_line_sequence: migration to 11.0

* Update sale_order_line_sequence.pot

* [MIG] sale_order_line_sequence v12

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update sale_order_line_sequence.pot

oca-transbot committed:
Update translation files

Updated by Update PO files to match POT (msgmerge) hook in Weblate.

mvrodriguez committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (8 of 8 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_line_sequence
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_line_sequence/es/

OCA-git-bot committed:
[ADD] icon.png

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (8 of 8 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_line_sequence
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_line_sequence/de/

OCA-git-bot committed:
[UPD] README.rst

liweijie0812 committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (8 of 8 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_order_line_sequence
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_order_line_sequence/zh_CN/

kongrattapong committed:
[IMP] sale_order_line_sequense: black, isort

kongrattapong committed:
[MIG] sale_order_line_sequence: Migration to 13.0

rajpatel2240-odoo committed:
[MIG] sale_order_line_sequence: Migration to 15.0

rousseldenis commented:
@patrickt-oforce Thanks for your contribution.

Can you follow migration guide https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0

(The commands are quite the same - inversing branch names)

patrickt-oforce commented:
> @patrickt-oforce Thanks for your contribution.
> 
> Can you follow migration guide https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0
> 
> (The commands are quite the same - inversing branch names)
Hi @rousseldenis, sorry for my mistakes but i've followed literraly the guide you suggested me. What means with `inversing branch name`?

rousseldenis commented on sale_order_line_sequence/__manifest__.py:
```diff
@@ -1,12 +1,16 @@
+    "version": "14.0.1.0.0",
+    "author": "Eficent, "
+    "Serpent CS, "
+    "Openforce srls, "
```
Migration does not allow you to put yourself as author.

Instead put a line in readme/CONTRIBUTORS.rst

patrickt-oforce replied:
Ok sorry. I'm fix this immediatly

patrickt-oforce replied:
@rousseldenis i need to remove the copyright line added?

rousseldenis commented:
> > @patrickt-oforce Thanks for your contribution.
> > Can you follow migration guide https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0
> > (The commands are quite the same - inversing branch names)
> > Hi @rousseldenis, sorry for my mistakes but i've followed literraly the guide you suggested me. What means with `inversing branch name`?

Read especially the 'Full process' part. You need to have first a IMP commit with pre-commit changes before migration one.

![image](https://user-images.githubusercontent.com/19529533/158988425-c6330e9e-dca8-4052-b44f-d97b38d62106.png)

patrickt-oforce commented:
> > > @patrickt-oforce Thanks for your contribution.
> > > Can you follow migration guide https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0
> > > (The commands are quite the same - inversing branch names)
> > > Hi @rousseldenis, sorry for my mistakes but i've followed literraly the guide you suggested me. What means with `inversing branch name`?
> 
> Read especially the 'Full process' part. You need to have first a IMP commit with pre-commit changes before migration one.
> 
> ![image](https://user-images.githubusercontent.com/19529533/158988425-c6330e9e-dca8-4052-b44f-d97b38d62106.png)

Thnak you for advice, can i rebase my history, having that as first commit?

rousseldenis commented:
> > > > @patrickt-oforce Thanks for your contribution.
> > > > Can you follow migration guide https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0
> > > > (The commands are quite the same - inversing branch names)
> > > > Hi @rousseldenis, sorry for my mistakes but i've followed literraly the guide you suggested me. What means with `inversing branch name`?
> > 
> > 
> > Read especially the 'Full process' part. You need to have first a IMP commit with pre-commit changes before migration one.
> > ![image](https://user-images.githubusercontent.com/19529533/158988425-c6330e9e-dca8-4052-b44f-d97b38d62106.png)
> 
> Thnak you for advice, can i rebase my history, having that as first commit?

No, let as is. For the next time :-)

rousseldenis commented:
> > > > > @patrickt-oforce Thanks for your contribution.
> > > > > Can you follow migration guide https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0
> > > > > (The commands are quite the same - inversing branch names)
> > > > > Hi @rousseldenis, sorry for my mistakes but i've followed literraly the guide you suggested me. What means with `inversing branch name`?
> > > 
> > > 
> > > Read especially the 'Full process' part. You need to have first a IMP commit with pre-commit changes before migration one.
> > > ![image](https://user-images.githubusercontent.com/19529533/158988425-c6330e9e-dca8-4052-b44f-d97b38d62106.png)
> > 
> > 
> > Thnak you for advice, can i rebase my history, having that as first commit?
> 
> No, let as is. For the next time :-)

That is done to separate changes that are real migration ones than cosmetics ones to easy review.

patrickt-oforce commented:
> > > > > @patrickt-oforce Thanks for your contribution.
> > > > > Can you follow migration guide https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0
> > > > > (The commands are quite the same - inversing branch names)
> > > > > Hi @rousseldenis, sorry for my mistakes but i've followed literraly the guide you suggested me. What means with `inversing branch name`?
> > > 
> > > 
> > > Read especially the 'Full process' part. You need to have first a IMP commit with pre-commit changes before migration one.
> > > ![image](https://user-images.githubusercontent.com/19529533/158988425-c6330e9e-dca8-4052-b44f-d97b38d62106.png)
> > 
> > 
> > Thnak you for advice, can i rebase my history, having that as first commit?
> 
> No, let as is. For the next time :-)

Thank you, but i've restarted migration as suggested

patrickt-oforce committed:
[IMP] sale_order_line_sequence: black, isort, prettier

patrickt-oforce committed:
[ADD] sale_order_line_sequence: add contributor

JuanyDForgeflow reviewed (approved):
Functional Review LGTM

AaronHForgeFlow reviewed (commented):
Functional :+1:  Can you check travis? It is other module, maybe a rebase should suffice to fix it

patrickt-oforce commented:
> Functional +1 Can you check travis? It is other module, maybe a rebase should suffice to fix it

Hi @AaronHForgeFlow, i've seen travis output, but the error is referred to a test failed in a module that i've not touched. My effort is to backport the module to a previous version. What do you mean with: "maybe a rebase should suffice to fix it"?

AaronHForgeFlow commented:
@patrickt-oforce , I mean that maybe the root cause of the error is now fixed in the base branch. You can just rebase your branch and perhaps it will just pass the checks. I think it worth a try.

patrickt-oforce commented:
> @patrickt-oforce , I mean that maybe the root cause of the error is now fixed in the base branch. You can just rebase your branch and perhaps it will just pass the checks. I think it worth a try.

Hi @AaronHForgeFlow , i've rebased the branch at actual 14.0 HEAD

patrickt-oforce commented:
Need to rebase, last commit?

AaronHForgeFlow commented:
@patrickt-oforce It it fine now I think. Tests are failing because of another module, "sale_by_packaging" I wonder if this module is not compatible with that one, or it is just an unrelated error :S

MiquelRForgeFlow commented:
Superceded by https://github.com/OCA/sale-workflow/pull/2104.

HaraldPanten commented:
Superseded by https://github.com/OCA/sale-workflow/pull/2104
